app/src/features/pricing/components/PricingTable/index.tsx

```diff
@@ -0,0 +1,253 @@
+            >
+              Renew
+            </RQButton>
+            {<Tag className="current-plan">Expired</Tag>}
+          </>
+        );
+      }
+
+      if (planName === PRICING.PLAN_NAMES.ENTERPRISE) {
+        return (
+          <RQButton onClick={() => setIsContactUsModalOpen(true)} type="primary">
+            Contact us
+          </RQButton>
+        );
+      }
+
+      if (product === PRICING.PRODUCTS.SESSION_REPLAY) {
+        if (planName === PRICING.PLAN_NAMES.FREE) {
+          return <div className="current-pricing-plan-tag">Current Plan</div>;
+        }
+
+        return (
+          <RQButton onClick={() => setIsContactUsModalOpen(true)} type="primary">
+            Contact us
+          </RQButton>
+        );
+      }
+
+      return (
+        <RQButton
+          onClick={() =>
+            isOpenedFromModal
+              ? handleOnSubscribe(planName)
+              : dispatch(
+                  actions.toggleActiveModal({
+                    modalName: "pricingModal",
+                    newValue: true,
+                    newProps: { selectedPlan: planName, workspace: workspaceToUpgrade },
+                  })
+                )
+          }
+          disabled={userPlanName === PRICING.PLAN_NAMES.PROFESSIONAL}
+          type="primary"
+        >
+          Upgrade now
+        </RQButton>
+      );
+    },
+    [product, user, workspaceToUpgrade, dispatch, handleOnSubscribe, isOpenedFromModal]
+  );
+
+  const renderFeaturesListHeader = (planName: string) => {
+    return (
+      <Row className="pro-basic-feature-title text-left">
+        {planName === PRICING.PLAN_NAMES.FREE && (
+          <Col>
+            <span>
+              All you need
+              <img src={underlineIcon} alt="highlight" />
+            </span>{" "}
+            to get started
+          </Col>
+        )}
+        {planName !== PRICING.PLAN_NAMES.FREE && (
+          <Col>
+            <span>
+              Everything <img src={underlineIcon} alt="highlight" />
+            </span>{" "}
+            in{" "}
+            {planName === PRICING.PLAN_NAMES.BASIC || product === PRICING.PRODUCTS.SESSION_REPLAY
+              ? PRICING.PLAN_HEADERS[PRICING.PLAN_NAMES.FREE as keyof typeof PRICING.PLAN_HEADERS]
+              : planName === PRICING.PLAN_NAMES.PROFESSIONAL
+              ? PRICING.PLAN_HEADERS[PRICING.PLAN_NAMES.BASIC as keyof typeof PRICING.PLAN_HEADERS]
+              : PRICING.PLAN_HEADERS[PRICING.PLAN_NAMES.PROFESSIONAL as keyof typeof PRICING.PLAN_HEADERS]}{" "}
+            plan, and
+          </Col>
+        )}
+      </Row>
+    );
+  };
+
+  return (
+    <>
+      <Row wrap={false} className="pricing-table">
+        {Object.entries(PricingFeatures[product]).map(([planName, planDetails]) => {
+          const planPrice =
+            PricingPlans[planName as keyof typeof PricingPlans]?.plans[
+              duration as keyof typeof PricingPlans[keyof typeof PricingPlans]["plans"]
```

Comment:
Have a type or enum for plan names and use that for `PricingPlans` and `PricingFeatures` with this we will have a single source for all the plan names or try to avoid this.